### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by naufaldsg**
- create "logo" folder
- add license and logo file
- add new logo to readme file
- add contribution on readme file

### CODE REVIEW

File: logo/LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
```
[the remark this answers is not in the document]
Comment (naufaldsg) [reply logo/LICENSE]: Deleted license, it make confused as you said.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+<img align="right" width="400" alt="Share to Clipboard" src="https://raw.githubusercontent.com/naufaldsg/share_to_clipboard/master/logo/sharetoclipboard-logotype-01.png">
```
[the remark this answers is not in the document]
Comment (naufaldsg) [reply README.md]: Edited. :)

**Comment by tasn** [comment]
Hey,

Awesome, thanks! Still missing the requests from the comment itself (the non-inline requests). Take another look at my last comment.

Thanks! I'm excited for this brush up. Never really liked the old logo, and this one is awesome. :)

**Comment by naufaldsg** [comment]
All done, can you check this [folder](https://drive.google.com/open?id=1d7npg_YIUvy9u9WS0ItTFIov1d1KHeoJ), whether the contents are in accordance with your request?

Icon made by material design guildelines
share-08

**Comment by tasn** [comment]
Almost, but not quite. The correct one would be the bottom left, but the shadowing is wrong. The shadowing there is as if the light is coming from the front (essentially the viewer's eyes), while in material the shadow should come from the top left. Additionally, the shadow is much much more subtle in material. Take another look at https://material.io/design/iconography/product-icons.html#specs

> 
> Shadows
> 
> A contact shadow is a result of the virtual light from above. A soft shadow surrounds a Material element lightly on the top and left,...
> Read More
> Contact shadow
> 
> A contact shadow is a result of the virtual light from above. A soft shadow surrounds a Material element lightly on the top and left, and with slightly more emphasis below and to the right of the element. The shadow is contained within the icon’s background silhouette.
> Drop shadow metrics
> 
> 
> Mode: Normal
> 
> Opacity: 20%
> 
> X Offset: 4dp
> 
> Y Offset: 4dp
> 
> Blur: 4dp
> 
> Color: Refer to Tint, shade and shadow values

**Comment by naufaldsg** [comment]
I've created based on your link give above. Sorry I misunderstood what you said, are you mean like this one?

share-09

**Comment by tasn** [comment]
No, the inside of the square is fine, revert it to how it was. I'm talking about the outside of the square.
Take another look at the icon here: https://play.google.com/store/apps/details?id=com.tengu.musicstudio
Do you see the shadowing outside the square? I'm not even sure I did it perfectly there, but I think that's more inline of how it should be.

**Comment by tasn** [comment]
I think your axis are not the same as material. The left one is the correct one. :)
The shadow looks a bit too large (look at your phone, you'll see plenty of examples). Maybe double-check your units, or worst case, cut it in half and I think it should be "good enough".

**Comment by naufaldsg** [comment]
I just want to clarify this, [material](https://material.io/design/iconography/product-icons.html#specs) said axis (x,y) is 4,4. This is right one. The left one is (4,-4). But, its ok for me. 😄 
screenshot_414


Well, this ok for you?
share-11

**Comment by tasn** [comment]
New one looks good, yeah!

As for your comment about the axis: some people/programs put the origin (0,0) at the top left corner, some at the bottom left. You're just not using the same axis system as material so you had to invert it. The new one is the correct one. Origin in material is top left.

**Comment by naufaldsg** [comment]
Yeaayy! thanks.
You can download logo source on the [folder](https://drive.google.com/open?id=1d7npg_YIUvy9u9WS0ItTFIov1d1KHeoJ) :)

**Comment by tasn** [comment]
Awesome, thanks!

One more thing left from my original review:
> 4. Please generate the png in the same sizes as the pngs in app/res/mipmap*. This will make it much easier for me to upload the app after merging this PR.

Could you please also do that?

Thanks!

**Comment by naufaldsg** [comment]
Its done, all files on drive, you can check it on LOGO SIZE folder. ;)

**Comment by tasn** [comment]
Could you please also put the files in the right place in the PR?

**Comment by tasn** [comment]
Please just push when you're ready, and let me know here. You're notification-spamming me. I'm getting an email once a minute.

**Comment by naufaldsg** [comment]
Sorry for spamming you, I cant use git command actually. I just modified and upload it via github.com. Im so sorry.

All done.

**Comment by tasn** [comment]
New version is up on play and should be available soon.
F-droid version will be built in the next few days.

**Pull request merged**